client/js/components/procedure/SegmentsList/SegmentsList.vue

```diff
@@ -558,6 +563,9 @@ export default {
         })
         .finally(() => {
           this.isLoading = false
+          this.$nextTick(() => {
+            this.$refs.imageModal.addClickListener(this.$refs.dataTable.$el.querySelectorAll('img'))
+          })
```

Comment:
I am not so happy with this. but I don't have a better solution.

Reply:
Me neither, but I didn't have another idea